userspace/libsinsp/ifinfo_test.cpp

```diff
@@ -69,7 +70,7 @@ void convert_to_string(char* dest, uint32_t addr)
 
 #define EXPECT_ADDR_EQ(dotted_notation,addr) {\
 	char buf[17];\
-	convert_to_string(buf,addr);\
+	convert_to_string(buf,17,addr);\
```

Comment:
Maybe we can use `sizeof(buf)` here? For both readability and coherence with the rest of the changes.
```suggestion
	convert_to_string(buf, sizeof(buf), addr);\
```

Reply:
of course!